src/app/components/ATIAnalytics/atiUrl/index.ts

```diff
@@ -417,3 +417,112 @@ export const buildATIEventTrackUrl = ({
     eventTrackingBeaconValues,
   )}&type=AT`;
 };
+
+export const buildReverbAnalyticsModel = ({
+  appName,
+  campaigns,
+  categoryName,
+  contentId,
+  contentType,
+  language,
+  ldpThingIds,
+  ldpThingLabels,
+  libraryVersion,
+  pageIdentifier,
+  pageTitle,
+  platform,
+  previousPath,
+  producerName,
+  origin,
+  nationsProducer,
+  statsDestination,
+  timePublished,
+  timeUpdated,
+}: ATIPageTrackingProps) => {
+  const href = getHref(platform);
+  const referrer = getReferrer(platform, origin, previousPath);
+
+  const aggregatedCampaigns = (Array.isArray(campaigns) ? campaigns : [])
+    .map(({ campaignName }) => campaignName)
+    .join('~');
+
+  const hashedId = getAtUserId();
+
+  const reverbVariables = {
+    params: {
+      page: {
+        contentId,
+        contentType,
+        destination: statsDestination,
+        name: pageIdentifier,
+        producer: producerName,
+        additionalProperties: {
+          app_name: platform === 'app' ? `${appName}-app` : appName,
+          app_type: getAppType(platform),
+          content_language: language,
+          product_platform: onOnionTld() ? 'tor-bbc' : null,
+          referrer_url:
+            referrer && encodeURIComponent(encodeURIComponent(referrer)),
+          x5: href && encodeURIComponent(encodeURIComponent(href)),
+          x8: libraryVersion,
+          x9: sanitise(pageTitle),
+          x10: nationsProducer && nationsProducer,
+          x11: timePublished,
+          x12: timeUpdated,
+          x13: ldpThingLabels,
+          x14: ldpThingIds,
+          x16: aggregatedCampaigns,
+          x17: categoryName,
+          x18: isLocServeCookieSet(),
+        },
+      },
+      user: {
+        hashedId,
+        isSignedIn: !!hashedId,
+      },
+    },
+    eventName: 'pageView',
+  };
+
+  return reverbVariables;
+};
+
+export const buildReverbPageSectionEventModel = ({
+  pageIdentifier,
+  producerName,
+  statsDestination,
+  componentName,
+  campaignID,
+  format,
+  type,
+  advertiserID,
+  url,
+}: ATIEventTrackingProps) => {
+  const eventPublisher = type === 'view' ? 'ati' : 'atc';
+
+  return {
+    params: {
+      page: {
+        destination: statsDestination,
+        name: pageIdentifier,
+        producer: producerName,
+        additionalProperties: {
+          [eventPublisher]: getEventInfo({
+            campaignID,
+            componentName,
+            format,
+            pageIdentifier,
+            advertiserID,
+            url,
+          }),
+          type: 'AT',
+        },
+      },
+      user: {
+        hashedId: getAtUserId(),
+        isSignedIn: false,
```

Comment:
Do we need the same logic as above (lines 480-481)?

Reply:
Thanks for highlighting this Karina.
I reckon we should. Let me take a look at the existing logic
to confirm what informed this.

Reply:
I've set this to `false` in both [buildReverbAnalyticsModel](https://github.com/bbc/simorgh/pull/12189/files#diff-0f5af613c62544299f969703b1a45445c249cfd998860abc4765113c77c27118R483) and [buildReverbPageSectionEventModel](https://github.com/bbc/simorgh/pull/12189/files#diff-0f5af613c62544299f969703b1a45445c249cfd998860abc4765113c77c27118R533).
This is informed by the documentation [here](https://navigation.api.bbci.co.uk/docs/bbcpage-and-bbcuser.md#:~:text=determining%20the%20country.-,isSignedIn(),-None.).
I've checked the beacons and confirmed that `idclient` is still reported in  beacons
fired to ATI/Piano.

Reply:
When I was working on my personalisation project, I had set it to be `!!hashedId` - see https://github.com/bbc/simorgh/pull/11353#discussion_r1572304765

@gavinspence is this something we would like to put in place now, or shall we leave it for later?

Reply:
Thanks for the context. I've responded in the thread in [11353](https://github.com/bbc/simorgh/pull/11353#discussion_r1920299089) for continuity.

I'll hold off merging pending steer from @gavinspence.

Reply:
That's fine @alex-magana, thanks for the detailed response and for following up with Product.